models/cow_protocol/ethereum/cow_protocol_ethereum_solvers.sql

```diff
@@ -101,7 +101,36 @@ known_solver_metadata (address, environment, name) as (
                  ('0x84e5c8518c248de590d5302fd7c32d2ae6b0123c', 'service', 'Withdraw'),
                  ('0xa03be496e67ec29bc62f01a428683d7f9c204930', 'service', 'Withdraw'),
                  ('0x2caef7f0ee82fb0abf1ab0dcd3a093803002e705', 'test', 'Test Solver 1'),
-                 ('0x56d4ed5e49539ebb1366c7d6b8f2530f1e4fe753', 'test', 'Test Solver 2')
+                 ('0x56d4ed5e49539ebb1366c7d6b8f2530f1e4fe753', 'test', 'Test Solver 2'),
+                 ('0x4F422556FaD14720B2691359fAd0C0F6B1B39113', 'prod', 'Naive'),
+                 ('0x96a3B53197eA4941482b142F5036E6432086Da8a', 'prod', 'Baseline'),
+                 ('0x8366812163991Dd7a54A0ec6191e5E5961Db845b', 'prod', 'Gnosis_1inch'),
+                 ('0x7AC1E2F8593c096Da0A94B71b6f70ca81b6b86F0', 'prod', 'Gnosis_ParaSwap'),
+                 ('0x8419FF1bCBCe6f0233cf3460B28c32b6951Fc457', 'prod', 'Gnosis_0x'),
+                 ('0xc20e52dDaFB05c67D78A440f2eAe344EB230A3dA', 'prod', 'Gnosis_BalancerSOR'),
+                 ('0x511d452b738b3f1aDA0E74e7A3412F5D975FC548', 'prod', 'Otex'),
+                 ('0x84d143682764DD1f42C4de763262107BD90c4F42', 'prod', 'PLM'),
+                 ('0x31A9Ec3A6E29039C74723E387DE42b79E6856FD8', 'prod', 'Laertes'),
+                 ('0x43872b55A12E087935765611851E94e3f0a79249', 'prod', 'Seasolver'),
+                 ('0x1e8D9a45175B2a4122F7827ce1eA3B08327b2ba0', 'prod', 'Quasilabs'),
+                 ('0xD8b9c8e1a94baEAaf4D1CA2C45723eb88236130E', 'prod', 'Raven'),
+                 ('0x0C60276BeaDc5BA35007661A89E0d5E7476523f8', 'prod', 'Barter'),
+                 ('0xC1624D29b82314Cd5cF52dEB293c87794bFAa9f0', 'barn', 'Legacy'),
+                 ('0x9AF3E1C8257557E2D70074fa03317F1A11595d02', 'barn', 'Naive'),
+                 ('0x542AAD0402F973a6FCFbf5d60dfC1b0C4233118c', 'barn', 'Baseline'),
+                 ('0x340c0C9E87C1E1ae5677506412e988748d8417ce', 'barn', 'Gnosis_1inch'),
+                 ('0x641830bD5E9F283e8f26c60846195e2201dFD09F', 'barn', 'Gnosis_ParaSwap'),
+                 ('0xeF7D51bD026E60ca75c2155419F29C0d31a6611C', 'barn', 'Gnosis_0x'),
+                 ('0x31Ac5E51a168B5179c703f3b05F120748C8c7c88', 'barn', 'Gnosis_BalancerSOR'),
+                 ('0xe374f64513f2432BcbF964B5ab84bD350D1FF222', 'barn', 'Otex'),
+                 ('0x3cdC01749eEEb4A26b1a9a9611328F232bE06be7', 'barn', 'PLM'),
+                 ('0xDE478f29C9566499f741cBF91CB068F1C2614B69', 'barn', 'Laertes'),
+                 ('0xD01BA5b3C4142F358EfFB4d6Cb44A11E31600330', 'barn', 'Seasolver'),
+                 ('0xF330Ee61d01ef1AC9FFA64662263e5E2DE93bbdE', 'barn', 'Quasilabs'),
+                 ('0x3C3513c88bD7A919Cb732F777Cd80cd773Beb011', 'barn', 'Raven'),
+                 ('0xAC0128ACC5c15945aB4E81F09f58CF05ec1844Ed', 'barn', 'Barter'),
+                 ('0xd3eEc78a89B04f95965927F408F07C13996B8378', 'barn', 'PropellerHeads'),
+                 ('0x452d604f08affFc4E87E74e3279BdBdeCeD07232', 'prod', 'PropellerHeads')
```

Comment:
Should `prod PropellerHeads` be moved to the end of the `prod` list?

Reply:
Reorder the solvers in the list to be grouped like before.